compiler/qsc_codegen/src/qsharp.rs

```diff
@@ -473,7 +473,7 @@ impl<W: Write> Visitor<'_> for QSharpGen<W> {
             }
             ExprKind::Field(record, name) => {
                 self.visit_expr(record);
-                self.write("::");
+                self.write(".");
```

Comment:
similar to comment on AST, is it possible this one should stay `::` since new field accessors are handled by the path segments code down below?

Reply:
See my response to the other comment: https://github.com/microsoft/qsharp/pull/1636#discussion_r1653333253